CHANGES_UNRELEASED.md

```diff
@@ -19,7 +19,26 @@ Use the template below to make assigning a version number during the release cut
 
 -->
 
+## Android
+
+### ⚠️ Breaking Changes ⚠️
+  - Many error classes have been renamed from `FooError` or `FooErrorException` to just `FooException`,
+    to be more in keeping with Java/Kotlin idioms.
+
+## Logins
+
+### ⚠️ Breaking Changes ⚠️
+  - Members of the `LoginsStoreError` enum no longer have a `message` property giving a detailed error message.
+    It is now just a plain enum with no associated data.
```

Comment:
@skhamis points out that this isn't going to work, because we depend on these error messages over in [the a-c wrapper](https://github.com/mozilla-mobile/android-components/blob/71e124a5dbafffb800168ea615fdbc54d7398f50/components/service/sync-logins/src/main/java/mozilla/components/service/sync/logins/DefaultLoginValidationDelegate.kt#L85). We'll need to bring them back somehow.

Reply:
Another option is to push forward with this and create the "fix" PR in a-c to take in the new version of a-s using enums instead of message but indeed would require more planning with the a-c team than we might want with just a uniffi ver bump